script/migrate-test-e2e.js

```diff
@@ -1,5 +1,11 @@
 import chalk from "chalk";
 import { $, execaCommand } from "execa";
+import { createVitest } from "vitest/node";
+
+const vitest = await createVitest("test", {
+	include: [new URL("./verify-changes.test.ts", import.meta.url).pathname],
+	watch: false,
+});
```

Comment:
[Praise] Ha, I forgot Vitest has a standalone `createVitest` method. Nifty usage of it!